packages/gatsby-theme-docs/package.json

```diff
@@ -43,8 +43,8 @@
     "@emotion/react": "^11.10.5",
     "@emotion/server": "^11.10.0",
     "@emotion/styled": "^11.10.5",
-    "@fontsource/roboto": "5.0.2",
-    "@fontsource/roboto-mono": "5.0.2",
+    "@fontsource/inter": "^5.0.5",
+    "@fontsource/roboto-mono": "^5.0.5",
```

Comment:
I'd keep the version locked, don't forget the font might be used from `docs` repo too and we should enforce the same version for both repos to avoid bundling the font twice